features/zones/PageNavigation.feature

```diff
@@ -33,8 +33,6 @@ Feature: Zones: Page navigation
       """
 
     When I visit the "/zones" URL
-    # This `cy.wait` stabilizes the test significantly. For some reason, it can happen that the subsequent navigation to via nav tab is never triggered.
-    Then I wait for 1000 milliseconds
```

Comment:
This `wait` is required because KTable:fetcher will overwrite this click (and subsequent router push) when we pass it our data. Avoiding this overwrite meant that the history e2e started failing. Which led me to fix that our history interactions. i.e. instead of avoiding adding possibly important history entries, we instead avoid adding non-important ones.